src/examples/mfem_common_plane.cpp

```diff
@@ -0,0 +1,429 @@
+  }
+private:
+  mfem::Vector accel;
+  mfem::Array<int> bc_vdofs;
+  bool first_step;
+
+  void SetHomogeneousBC(mfem::Vector& dxdt) const
+  {
+    for (auto bc_vdof : bc_vdofs)
+    {
+      dxdt[bc_vdof] = 0.0;
+    }
+  }
+};
+
+/// Explicit solid mechanics update (lumped mass, optional damping)
+class ExplicitMechanics : public mfem::SecondOrderTimeDependentOperator
+{
+public:
+  ExplicitMechanics(
+    mfem::ParFiniteElementSpace& fespace, 
+    mfem::Coefficient& rho,
+    mfem::Coefficient& lambda,
+    mfem::Coefficient& mu,
+    std::unique_ptr<mfem::ParBilinearForm> damping_ = nullptr
+  )
+  : elasticity { &fespace },
+    damping { std::move(damping_) },
+    inv_lumped_mass { fespace.GetVSize() }
+  {
+    mfem::ParBilinearForm mass { &fespace };
+    mass.AddDomainIntegrator(new mfem::VectorMassIntegrator(rho));
+    mass.Assemble();
+    mfem::Vector ones {fespace.GetVSize()};
+    ones = 1.0;
+    mass.SpMat().Mult(ones, inv_lumped_mass);
+    mfem::Vector mass_true(fespace.GetTrueVSize());
+    const Operator& P = *fespace.GetProlongationMatrix();
+    P.MultTranspose(inv_lumped_mass, mass_true);
+    for (int i {0}; i < mass_true.Size(); ++i)
+    {
+      mass_true[i] = 1.0 / mass_true[i];
+    }
+    P.Mult(mass_true, inv_lumped_mass);
+
+    elasticity.AddDomainIntegrator(new mfem::ElasticityIntegrator(lambda, mu));
+    elasticity.Assemble();
+  }
+
+  void Mult(
+    const mfem::Vector& u,
+    const mfem::Vector& dudt,
+    mfem::Vector& a
+  ) const override
+  {
+    mfem::Vector f { u.Size() };
+    f = 0.0;
+
+    mfem::Vector f_int { f.Size() };
+    elasticity.Mult(u, f_int);
+    f.Add(-1.0, f_int);
+
+    if (damping)
+    {
+      damping->Mult(dudt, f_int);
+      f.Add(-1.0, f_int);
+    }
+
+    // sum forces over ranks
+    auto& fespace = *elasticity.ParFESpace();
+    const Operator& P = *fespace.GetProlongationMatrix();
+    mfem::Vector f_true {fespace.GetTrueVSize()};
+    P.MultTranspose(f, f_true);
+    P.Mult(f_true, f);
+
+    // force already summed over ranks
+    f.Add(1.0, tribol::getMfemResponse(0));
+
+    for (int i {0}; i < inv_lumped_mass.Size(); ++i)
+    {
+      a[i] = inv_lumped_mass[i] * f[i];
+    }
+  }
+
+private:
+  mfem::ParBilinearForm elasticity;
+  std::unique_ptr<mfem::ParBilinearForm> damping;
+  mfem::Vector inv_lumped_mass;
+};
+
+int main( int argc, char** argv )
+{
+  // initialize MPI
+  MPI_Init( &argc, &argv );
+  int np, rank;
+  MPI_Comm_size(MPI_COMM_WORLD, &np);
+  MPI_Comm_rank(MPI_COMM_WORLD, &rank);
+
+#ifdef TRIBOL_USE_UMPIRE
+  umpire::ResourceManager::getInstance();  // initialize umpire's ResouceManager
+#endif
+
+  // initialize logger
+  axom::slic::SimpleLogger logger;
+  axom::slic::setIsRoot(rank == 0);
+
+  // command line options
+  // number of times to uniformly refine the serial mesh before constructing the
+  // parallel mesh
+  int ref_levels = 2;
+  // polynomial order of the finite element discretization
+  int order = 1;
```

Comment:
Was this example or its test run with order > 1 and on multiple ranks?

Reply:
Yes, but the results aren't consistent since we need to parallel LOR transfer MFEM PR to make it work correctly.  The feature has been marked "experimental" in this PR.